utilities/tez-ui/README.md

```diff
@@ -25,9 +25,12 @@ You can use a pair of AWS access key and secret key, or temporary AWS credential
 1. Set a few environment variables relevant to your job.
 
 ```shell
-export S3_LOG_URI=s3://${S3_BUCKET}/hive-logs
 export APPLICATION_ID=001122334455
 export JOB_RUN_ID=667788990011
+SERVERLESS_EMR_S3_LOG_BUCKET=$S3_BUCKET
```

Comment:
Seems redundant to set this variable here when it's just being set to `$S3_BUCKET`.